unit_tests/server/test_server_request_responder_main.cpp

```diff
@@ -115,22 +115,33 @@ struct test_env
   test_thread_init_final m_thread_init_final;   // only to initialize/finalize cubthread
   std::array<test_conn, T_CONN_COUNT> m_conns;
   // responder must be destroyed before connections because responder handles connections in its dtor
-  server_request_responder<test_conn> m_rrh;
+  cubthread::entry_workpool *m_worker_pool;
+  std::unique_ptr<server_request_responder<test_conn>> m_rrh;
 
   test_env ()
   {
+    const size_t responder_thread_cnt = (size_t) prm_get_integer_value (PRM_ID_SCAL_PERF_PS_REQ_RESPONDER_THREAD_COUNT);
+    const size_t responder_max_task_cnt = (size_t) prm_get_integer_value (PRM_ID_SCAL_PERF_PS_REQ_RESPONDER_TASK_COUNT);
+
+    m_worker_pool = cubthread::get_manager ()->create_worker_pool (responder_thread_cnt, responder_max_task_cnt,
+		    "test_async_responder_workers", NULL, 1, false);
```

Comment:
I don't think it's needed. The context manager determines the `tran_index` and thread type of entries (workers). It may affect the work each cubrid thread does, but does not affect the `server_request_responder` itself. This unit test is for `server_request_responder`.